acinclude.m4

```diff
@@ -873,10 +871,7 @@ struct Library *SocketBase = NULL;
 #endif
 #define RECVCALLCONV PASCAL
 #else
-#ifdef HAVE_PROTO_BSDSOCKET_H
-#include <proto/bsdsocket.h>
-struct Library *SocketBase = NULL;
-#endif
+                      $curl_includes_bsdsocket
```

Comment:
Is this huge leading white space a mistake?

Reply:
Perhaps 😕. It is indented to match the level of the code below, but I didn't do that anywhere else, so I'll remove the white space on this.